tests/testthat/test-get_source_expressions.R

```diff
@@ -93,9 +93,12 @@ test_that("Multi-byte character truncated by parser is ignored", {
   skip_if_not_utf8_locale()
   # \U2013 is the Unicode character 'en dash', which is
   # almost identical to a minus sign in monospaced fonts.
-  with_content_to_parse("y <- x \U2013 42", {
-    expect_identical(error$message, "unexpected input")
-    expect_identical(error$column_number, 8L)
+  content <- "y <- x \U2013 42"
+  # message is like '<text>:1:8: unexpected invalid token\n1: ...'
```

Comment:
Maybe use rex with capture groups to auto-parse the base msg to expected `message` and `column`?

Reply:
I was tempted... at a certain point we're just copying the implementation straight to the test right?

how about (1) leaving literal 8 as the expected column # and (2) just checking grepl(actual, expected) to ensure we've pulled out any subset of the true error?